.test/environment.yaml

```
@@ -0,0 +1,6 @@
+channels:
+    - https://prefix.dev/conda-forge
+    - https://prefix.dev/bioconda
+    - nodefaults
+dependencies:
+    - snakemake ==9.5.1
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)